core/sql/regress/executor/EXPECTED130

```diff
@@ -124,7 +134,16 @@ inserted row13
 
 --- 3 row(s) selected.
 >>
->>
+>>values (filetolob('./myfile.txt'));
+
+*** ERROR[8442] Unable to access ExpLOB interface. Call to ExpLOB returned error LOB_PTR_ERROR(532). Error detail 0.
```

Comment:
On second thought it's not possible to distiguish the function being used in an input values clause Vs the values clause by itself at compiletime. So in line with other functions like queryid_extract, I can improve the error message so it's clear that the function is not being called on a valid LOB column. A new error 8434 can be introduced for this. 
>>values(filetolob('fffff'));

*** ERROR[8434] Invalid target column for LOB function. The column needs to be blob/clob type.
--- 0 row(s) selected.


>>values(stringtolob('gggggg'));
*** ERROR[8434] Invalid target column for LOB function. The column needs to be blob/clob type.
--- 0 row(s) selected.

>>values(buffertolob(LOCATION 6767676,size 5));

*** ERROR[8434] Invalid target column for LOB function. The column needs to be blob/clob type.
--- 0 row(s) selected.
>>